lib/handbook/addon/docs/components/ancestry-display/controller.ts

```diff
@@ -0,0 +1,38 @@
+import Controller from '@ember/controller';
+import EmberObject from '@ember/object';
+import { faker } from 'ember-cli-mirage';
+
+export default class DemoAncestryDisplay extends Controller {
+    parentNode: any;
+    childNode: any;
+    grandChildNode: any;
+    greatGrandChildNode: any;
+
+    init(this: DemoAncestryDisplay) {
+        super.init();
+        const parentNode = this.createNode('ezcux');
+        const childNode = this.createNode('ezcux0', { parent: parentNode });
+        const grandChildNode = this.createNode('ezcux1', { parent: childNode, root: parentNode });
+        const greatGrandChildNode = this.createNode('ezcux2', { parent: grandChildNode, root: parentNode });
+
+        this.setProperties({
+            parentNode,
+            childNode,
+            grandChildNode,
+            greatGrandChildNode,
+        });
+    }
+
+    createNode(id: string, extra?: object): any {
```

Comment:
Chatted with @brianjgeiger about this guy. The mirage `server` is available to you here sadly but you can access data in the [default scenario](https://github.com/CenterForOpenScience/ember-osf-web/blob/a6d08032087dd33e16a5a09ae9c5e5b36c3b221b/mirage/scenarios/default.ts).

I would suggest adding some nested nodes to the default scenario and just loading them via `store.load('node', 'GUID')`